INSTALL.md

```diff
@@ -1,143 +1,120 @@
+them.
 
-* `pisa ` directory of source code
-* `-e`  (editable) Installs from source located at `$PISA` (Installed directory - pisa) and  allows for changes to the source code within to be immediately propagated to your Python installation.
-* `-vvv` Be maximally verbose during the install. You'll see lots of messages, including warnings that are irrelevant, but if your installation fails, it's easiest to debug if you use `-vvv`
-* `[develop]` Specify optional dependency groups. You can omit any or all of these if your system does not support them or if you do not need them.
 
-### **Test your installation by running a simple script:**
+### Test PISA
 
-- Activate your python environment (`pisa_env`)
+ First activate your python environment (if not already active). Assuming you used Miniconda, you can do that with:
 
   ```bash
-  conda activate pisa_env
+  source PATH_TO_ANACONDA/miniconda3/bin/activate
+  conda activate NAME_OF_YOUR_PISA_ENV
   ```
-- Start python interpretator in terminal
 
-  ```bash
-  python
-  ```
-- Import `Pipline` modelue and `matplotlib.pyplot`
+#### Running a simple script
 
-  ```python
-  from pisa.core import Pipeline
-  import matplotlib.pyplot as plt
-  ```
-  ```
-  << PISA is running in single precision (FP32) mode; numba is running on CPU (single core) >>
-  ```
-- Load an example pipeline
+Running the following command should create a folder in `/tmp` containing a pdf with output maps for different neutrino types and interactions.
 
-  ```python
-  template_maker = Pipeline("settings/pipeline/osc_example.cfg")
-  ```
-- Run the pipeline
+```bash
+$PISA/pisa/core/distribution_maker.py --pipeline settings/pipeline/IceCube_3y_neutrinos.cfg --outdir /tmp/pipeline_output --pdf
+```
 
-  ```python
-  template_maker.run()
-  ```
-- Get the oscillation probabilities as a map
+#### Unit Tests
 
-  ```python
-  outputs = template_maker.data.get_mapset('prob_mu')
-  ```
-- Plot the oscillogram
+Throughout the codebase there are `test_*.py` files and `test_*` functions within various `*.py` files that represent unit tests.
+Unit tests are designed to ensure that the basic mechanisms of objects' functionality work.
 
-  ```python
-  fig, axes = plt.subplots(figsize=(18, 5), ncols=3)
-  outputs['nue_cc'].plot(ax=axes[0], cmap='RdYlBu_r', vmin=0, vmax=1);
-  outputs['numu_cc'].plot(ax=axes[1], cmap='RdYlBu_r', vmin=0, vmax=1);
-  outputs['nutau_cc'].plot(ax=axes[2], cmap='RdYlBu_r', vmin=0, vmax=1);
-  ```
 
-### Instructions to install PISA on Madison working group servers cobalts (current guide from August 2023):
+## Instructions to install PISA using cvmfs and virtual environment
 
 Assuming you already are in the directory where you want to store fridge/pisa source files and the python virtualenv and build pisa. You also need access to github through your account.
```

Comment:
Let's not deal with fridge here. For example, we just need the name of that repo to change for these instructions to be out of sync.